src/observable/fromEvent.ts

```diff
@@ -1,40 +1,60 @@
 import {Observable} from '../Observable';
+import {isEventTarget} from '../util/isEventTarget';
+import {isHTMLCollection} from '../util/isHTMLCollection';
+import {isNodeList} from '../util/isNodeList';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 import {Subscription} from '../Subscription';
 import {Subscriber} from '../Subscriber';
 
+export type EventEmmitterLike1 = {
```

Comment:
@kwonoj 

As the ínterim summary of this discussion, I think these naming might be a bit clearly: `NodeStyleEventEmitter` & `jQueryStyleEventEmitter`. How about this?

As my personal thinking, I favor the convention `BarStyle` or `FooLike` to open a library which follows a style of jQuery.Event or Node's EventEmmitter.